lib/pg_bulkload--unpackaged.sql.in

```diff
@@ -1,4 +1,4 @@
-/* pg_bulkload/pg_bulkload--unpackaged--1.0.sql */
```

Comment:
Don't we need the file now?

Reply:
No, we don't need this file. I will delete it.